.travis.yml

```diff
@@ -41,6 +41,10 @@ install:
       fi
   - pip install -U tox twine wheel codecov
 
+addons:
+  hosts:
+    - xn--pxaix.localhost
```

Comment:
Thanks for the feedback @dangra . I don't know how to test this then.
Any ideas?